src/lazy/struct.rs

```diff
@@ -78,6 +77,28 @@ impl<'top, D: LazyDecoder> LazyStruct<'top, D> {
         }
     }
 
+    //TODO: Feature gate
+    pub fn lower(&self) -> LazyExpandedStruct<'top, D> {
+        self.expanded_struct
+    }
+    pub fn as_value(&self) -> LazyValue<'top, D> {
```

Comment:
The whole reader API will be behind a feature gate for the near future. However, I think `as_value` can be exposed when the reader API is because `LazyValue` and the `LazyList`/`LazyStruct/`LazySExp` types are all part of the reader API.